functests/utils/images/images.go

```diff
@@ -0,0 +1,50 @@
+package images
+
+import (
+	"fmt"
+	"log"
+	"os"
+)
+
+var registry string
+var images map[string]imageLocation
+
+func init() {
+	registry = os.Getenv("IMAGE_REGISTRY")
+
+	images = map[string]imageLocation{
+		"sctp": {
+			name:    "sctptester",
+			registy: "quay.io/fpaoline/",
+			version: "v1.0",
+		},
+		"testutils": {
+			name:    "cnftest-utils",
+			registy: "quay.io/fpaoline/",
+			version: "v1.0",
+		},
+		"stresser": {
+			name:    "stresser",
+			registy: "quay.io/fpaoline/",
+			version: "v1.0",
+		},
+	}
+}
+
+type imageLocation struct {
+	name    string
+	registy string
+	version string
+}
+
+// For returns the image to be used for the given key
+func For(name string) string {
+	img, ok := images[name]
+	if !ok {
+		log.Printf("Image for %s not found", name)
```

Comment:
Should the if return as well? Otherwise it prints the log, but continues to the Sprintfs, right?

Reply:
Aw right, it was a last second addition for better feedback.
Thanks!

Reply:
Well technically it would have worked in any case, because `img` would contain zero-valued strings, but it was not intentional :-)

Reply:
Now, if an image is not found, I assume we will run into funny errors when trying to create a pod with an empty image field. I would prefer to catch this before creating the pod, by failing the test immediately.